src/components/DragDrop/DragDrop.module.css

```diff
@@ -169,15 +176,19 @@
   display: flex;
   justify-content: center;
   align-items: center;
-  opacity: 0;
   height: calc(100% - 2px);
   padding: var(--eds-size-1);
-  z-index: var(--eds-z-index-500);
+  z-index: var(--eds-z-index-400);
```

Comment:
Why was the z-index 500 before and now 400? Can you leave a comment for its justification

Reply:
To not overlay the dropdown menu in the project card